src/CustomDatePickerIOS/index.js

```diff
@@ -148,8 +162,9 @@ export default class CustomDatePickerIOS extends PureComponent {
       >
         <View style={[styles.datepickerContainer, datePickerContainerStyleIOS]}>
           {customTitleContainerIOS || titleContainer}
-          <View onStartShouldSetResponderCapture={this._handleUserTouchInit}>
-            <DatePickerIOS
+          <View onStartShouldSetResponderCapture={neverDisableConfirmIOS !== true ? this._handleUserTouchInit : null}>
```

Comment:
You should revert this to ` <View onStartShouldSetResponderCapture={this._handleUserTouchInit}>`

Reply:
Hmm any reason why we should do that?
People may want to disable the confirmation buttons even if they're not using a Custom date picker.

Can you think of any reasons to revert?

Reply:
Is there any reason to make this change? any scenario where we need this condition?

Although it doesn't generate any problems:
- neverDisableConfirmIOS feature works without this condition (when using default datepicker)
- The main problem I see is readability. It's easier to follow code if we leave this condition out
- "neverDisableConfirmIOS !== true" is confusing

So, can we leave it out?